### PR TITLE
Create destroy method for courses

### LINKED ISSUE #12: [1pt] Create controller action(destroy) for classes model
- Implement action to delete a class

### PR DESCRIPTION
**Pull request by elyor-doniyorov**
- Implement action to delete a course

### CODE REVIEW

File: app/controllers/api/v1/courses_controller.rb [hunk 1]
```diff
@@ -24,6 +24,11 @@ def create
     end
   end
 
+  def destroy
+    @course = course.find(params[:id])
+    @course.destroy
+  end
```
Comment (enkog) [inline app/controllers/api/v1/courses_controller.rb]: I will suggest you return something to indicate a successful deletion of the course.
You can add a `header :no_content` after destroying the course.

**Review by enkog — CHANGES_REQUESTED** [review]
♻️  **Required Changes**

Good job so far, @elyor-doniyorov 

You've done an excellent job developing the destroy action for the courses.

However, it would be nice to also consider making this little change.

Keep being an awesome developer 💻

File: app/controllers/api/v1/courses_controller.rb (hunk 1 above)
Comment (elyor-doniyorov) [reply app/controllers/api/v1/courses_controller.rb]: Hi @enkog,

Thanks for your detailed review! I did all of the required changes. You can review it now, please

**Review by enkog — APPROVED** [review]
### **APPROVED** ✔️ 

Good job implementing the change.

**Pull request merged**